src/Microsoft.Tye.Hosting/Model/Application.cs

```diff
@@ -139,7 +139,8 @@ private List<EffectiveBinding> ComputeBindings(Service service, string defaultHo
                 foreach (var b in s.Description.Bindings)
                 {
                     var protocol = b.Protocol;
-                    var host = b.Host ?? (service.Description.RunInfo is DockerRunInfo ? s.Description.Name : defaultHost);
+                    // Just ignored the binding host.
+                    var host = service.Description.RunInfo is DockerRunInfo ? s.Description.Name : defaultHost;
```

Comment:
Yeah this seems odd, when running locally, b.Host contains the parameter that is passed in for --host. Shouldn't that be used here?

Reply:
Yes, but doesn't work with that value.

Reply:
So is the host provided here supposed to just be for the dashboard or all backend services? I think that's where I'm personally confused.

Reply:
The value assigned to the `host` is used for "something" with the all backed services. The usage of value in `b.Host` creates a wrong configuration and can reach only one. That was the reason for tests failures.  

Ignoring the `b.Host` for the service conguration setup all "correctly" in local mode.